engines/scumm/scumm.cpp

```diff
@@ -2435,6 +2435,9 @@ void ScummEngine::scummLoop_handleSaveLoad() {
 
 			if (success && _saveTemporaryState && VAR_GAME_LOADED != 0xFF && _game.version <= 7)
 				VAR(VAR_GAME_LOADED) = 201;
+
+			if (!_saveTemporaryState)
+				_lastSaveTime = _system->getMillis();
```

Comment:
Previously, `_lastSaveTime` was also reset when loading a save. Now not anymore. Is that change intentional?

Reply:
This is indeed intentional (apparently my commit message isn't as unambiguous as I had hoped), but I'm not sure either if that is "correct"/OK (hence the PR) because I don't know how and when scripts actually use these opcodes.

Only v8 seems to have an opcode for loading, older versions down to v5 can can only save which is puzzing to me because that suggests that the user should be able to access these savegames, why else write them in the first place if scripts can't load them again? But as I mentioned this is currently not possible because the launcher/GMM doesn't list savegames with the temporary file extension.

My rationale was that since these savegames are not accessible by the user they should never interfere with our autosaves, not even on loading, which is why I didn't go with the more defensive approach (keep the instruction where it was and also reset on loading).

¯\\_(ツ)_/¯

Reply:
Oh, right. Your change looks good to me, though I'm not SCUMM engine expert.

Reply:
Thanks for looking at it!